dev-data/index.ts

```diff
@@ -30,18 +23,14 @@ export {
   RS_DOCS_EN_LINK,
   RS_DOCS_TELEGRAM_CHATS_LINK,
 } from './communication.data';
-
+export { type Benefit } from './benefit-mentorship.data';
```

Comment:
```suggestion
export type { Benefit } from './benefit-mentorship.data';
```

Reply:
@ansivgit We should probably add a corresponding eslint rule